installation/proc-install-rhdh-helm.adoc

```diff
@@ -27,17 +27,34 @@ For more information about creating a project in OpenShift, see link:https://doc
 . From the *Developer Catalog* panel, click *Helm Chart*.
 . Search for _Developer Hub_ in the search bar and select the *{product}* card.
 . Click *Create*.
-. Copy the OpenShift router host (for example: `apps.<clusterName>.com`) to *Root Schema* -> *global* -> *Shorthand for users who do not want to specify a custom HOSTNAME. Used ONLY with the DEFAULT upstream.backstage.appConfig value and with OCP Route enabled.*, and adjust the other values if needed.
+. Use one of the following views to edit the Helm configuration:
 
-. Alternatively, copy the OpenShift router host (for example: `apps.<clusterName>.com`) to `global.clusterRouterBase`, and adjust other values if needed, such as `global.clusterRouterBase: apps.example.com`. 
+** Using *Form view* 
++
+.. Copy the OpenShift router host (for example: `apps.<clusterName>.com`) and paste it in the *Root Schema* -> *global* -> *Shorthand for users who do not want to specify a custom HOSTNAME. Used ONLY with the DEFAULT upstream.backstage.appConfig value and with OCP Route enabled.* field, and edit the other values if needed.
+.. Click *Create* and wait for the database and {product} to start.
+
+** Using *YAML view* 
++
+.. Copy the OpenShift router host (for example: `apps.<clusterName>.com`) and paste it in the `global.clusterRouterBase` parameter value as shown in the following example:
++
+[source,yaml]
+----
+global:
+  auth:
+    backend:
+      enabled: true
+  clusterRouterBase: apps.<clusterName>.com
+  # other Red Hat Developer Hub Helm Chart configurations
+----
+.. Click *Create* and wait for the database and {product} to start.
 +
 --
-In the previous steps, the information about the host is copied, which is accessed by the {product-short} backend. 
+In both the views, the copied information about the host is accessed by the {product-short} backend. 
 
-When an OCP route is generated automatically, the host value for the route is inferred and the same host information is sent to the {product-short}. Also, if the {product-short} is present on a custom domain by setting the host manually using values, the custom host takes precedence.
+When an OCP route is generated automatically, the host value for the route is inferred and the same host value is sent to the {product-short}. Also, if you have set up the host manually to configure {product-short} on a custom domain, the custom host value for the route takes precedence.
```

Comment:
OCP is not an approved name. RHOCP is approved, but we generally use the long name.

Reply:
I think your PR is going to fix this issue. You have used the {ocp-short} attribute. I have tweaked the second sentence a bit for better clarity.